src/main/java/com/blamejared/crafttweaker/impl_native/event/entity/player/xp/ExpandLevelChangedEvent.java

```diff
@@ -0,0 +1,47 @@
+package com.blamejared.crafttweaker.impl_native.event.entity.player.xp;
+
+
+import org.openzen.zencode.java.ZenCodeType;
+
+import com.blamejared.crafttweaker.api.annotations.ZenRegister;
+import com.blamejared.crafttweaker_annotations.annotations.Document;
+import com.blamejared.crafttweaker_annotations.annotations.NativeTypeRegistration;
+
+import net.minecraftforge.event.entity.player.PlayerXpEvent.LevelChange;
+
+/**
+ * This event is fired when the player's level count is changed. If the amount
+ * is positive the levels are being added. If they are negative levels are
```

Comment:
> If they are negative levels are

shouldn't this be:
```
If they are negative, levels are
```

First read through of this I was confused as to what `negative levels` was.